views/index.jsx

```diff
@@ -44,13 +45,17 @@ function render({ slug, meta, html }) {
 			<PageBreadcrumb slug={slug} pageTitle={meta.h1} />
 			<ContainerWhenSubpage slug={slug}>
 				{safe(html)}
+
+				<Footer />
 			</ContainerWhenSubpage>
 		</DefaultLayout>;
 	}
 
 	return <DefaultLayout title={meta.title}>
 		<Container>
 			{safe(html)}
+
+			<Footer />
```

Comment:
this doesn't actually seem to be used anywhere (cf. my front-page confusion)

Reply:
You're right that this isn't being used anywhere because we always set `meta.h1` in all of our pages, so `if(meta.h1)` is never false. Don't know if we should just delete this conditional here?